doc/sonic-installer/SONiC_Image_Installation_Verification.md

```diff
@@ -0,0 +1,172 @@
+
+# 3 Configuration and Management Requirements
+
+- The option of platform check skip hould be added to sonic-installer help menu.
+  - Use `--skip-platform-check` option instead of `--force` as the later one has been used for image force installation which skips image secure check. 
+```
+admin@vlab-01:~$ sudo sonic-installer install --help
+Usage: sonic-installer install [OPTIONS] URL
+
+  Install image from local binary or URL
+
+Options:
+  -y, --yes
+  -f, --force, --skip-secure-check
+                                  Force installation of an image of a non-
+                                  secure type than secure running image
+  --skip-platform-check           Force installation of an image of a type       <======
+                                  which is not of the same platform
+  --skip_migration                Do not migrate current configuration to the
+                                  newly installed image
+  --skip-package-migration        Do not migrate current packages to the newly
+                                  installed image
+  --skip-setup-swap               Skip setup temporary SWAP memory used for
+                                  installation
+  --swap-mem-size INTEGER         SWAP memory space size  [default: (1024
+                                  MiB)] NOTE: this argument is mutually
+                                  exclusive with arguments: skip_setup_swap
+  --total-mem-threshold INTEGER   If system total memory is lower than
+                                  threshold, setup SWAP memory  [default:
+                                  (2048 MiB)] NOTE: this argument is mutually
+                                  exclusive with arguments: skip_setup_swap
+  --available-mem-threshold INTEGER
+                                  If system available memory is lower than
+                                  threhold, setup SWAP memory  [default: (1200
+                                  MiB)] NOTE: this argument is mutually
+                                  exclusive with arguments: skip_setup_swap
+  --help                          Show this message and exit.
+
+```
+
+# 4 Design and Flow Diagrams
+
+## 4.1 Flow Diagrams
+The following figure shows how to intall an image through SONiC CLI.
+```
++-----------------+  
+|   SONiC ClI     |            
+|                 <-----------+
+|  Image Install  |           |
++-----------------+           |
+         |                    |
+         V                    |
++-----------------+           |
+|      Image      |---->NO----+
+|                 |             +------------+
+|   Verification  |---->YES-----> Installing |
++-----------------+             +------------+
+```
+
+## 4.2 Design
+
+### 4.2.1 Generate platform list
+We need to generate a file `platforms_asic`. This file contains the list of supported platforms and is generated during the process of image build. The file is kept in built image.
```

Comment:
>platforms_asic [](http://example.com/codeflow?start=28&length=14)

Could you gave some example or snippet? One for broadcom image, and one for mellanox image.